bundles/org.openhab.binding.chatgpt/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -19,6 +19,26 @@
 				<label>API Key</label>
 				<description>API key to access the account</description>
 			</parameter>
+			<parameter name="apiUrl" type="text" required="false">
+				<label>API URL</label>
+				<description>The server API where to reach the AI service.</description>
+				<default>https://api.openai.com/v1/chat/completions</default>
+				<advanced>true</advanced>
+				<options>
+					<option value="https://api.openai.com/v1/chat/completions">https://api.openai.com/v1/chat/completions</option>
```

Comment:
@mudler What would be the suggested option here for LocalAI?

Reply:
For LocalAI would be enough to point it to the LocalAI service, so for instance, if you are running it locally on the `8080` port would be:

http://localhost:8080/v1/chat/completions

Reply:
Port 8080 is also used by openHAB, so a user running both on the same machine would need to use a non-default port for LocalAI... Sounds as if it is hence difficult to provide a suggestion here.

Reply:
@mudler Could you define an alternative port for LocalAI users to configure, if they are installing it on the same machine as openHAB ( and integrating with it)? If so, I would put that port as a value in here. Using `8080` is simply not working...

Reply:
@mudler I actually failed setting up LocalAI on a different port. I changed the `ADDRESS` parameter in the `.env` file, but it didn't have any effect...

Reply:
apologize, I missed this until now - did you tried with docker only?

```
docker run -p 9090:8080 -v $PWD/models:/models -ti --rm quay.io/go-skynet/local-ai:latest --models-path /models --context-size 700 --threads 4
```